ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py

```diff
@@ -1,184 +1,185 @@
             metadata = pcgts.get_Metadata()
             metadata.add_MetadataItem(
-                    MetadataItemType(type_="processingStep",
-                                     name=self.ocrd_tool['steps'][0],
-                                     value=TOOL,                                     
-                                     Labels=[LabelsType(#externalRef="parameters",
-                                                        Label=[LabelType(type_=name,
-                                                                         value=self.parameter[name])
-                                                               for name in self.parameter.keys()])]))
+                MetadataItemType(type_="processingStep",
+                                 name=self.ocrd_tool['steps'][0],
+                                 value=TOOL,
+                                 Labels=[LabelsType(  # externalRef="parameters",
+                                     Label=[LabelType(type_=name,
+                                                      value=self.parameter[name])
+                                            for name in self.parameter.keys()])]))
 
             page = pcgts.get_Page()
-            
-            page_image, page_xywh, page_image_info = self.workspace.image_from_page(page, page_id, feature_filter='dewarped', feature_selector='binarized') # images should be deskewed and cropped
+
+            page_image, page_xywh, _ = self.workspace.image_from_page(
+                page, page_id, feature_filter='dewarped', feature_selector='binarized')  # images should be deskewed and cropped
             if oplevel == 'page':
-                dataset = self.prepare_data(opt, page_image, path)
+                dataset = self.prepare_data(opt, page_image)
                 orig_img_size = page_image.size
-                self._process_segment(model, dataset, page, page_xywh, page_id, input_file, orig_img_size, n)
+                self._process_segment(
+                    model, dataset, page, page_xywh, page_id, input_file, orig_img_size, n)
             else:
-                regions = page.get_TextRegion() + page.get_TableRegion() #get all regions?
-                if not regions: 
+                regions = page.get_TextRegion() + page.get_TableRegion()  # get all regions?
+                if not regions:
                     LOG.warning("Page '%s' contains no text regions", page_id)
-                for (k, region) in enumerate(regions):
-                    region_image, region_xywh = self.workspace.image_from_segment(region, page_image, page_xywh)            
+                for _, region in enumerate(regions):
+                    region_image, region_xywh = self.workspace.image_from_segment(
+                        region, page_image, page_xywh)
                     # TODO: not tested on regions
                     # TODO: region has to exist as a physical file to be processed by pix2pixHD
```

Comment:
BTW we should check whether this `TODO` warning is correct. If these strange `pix2pixHD` data loader classes (which themselves seem to be wrappers around `torch.utils.data.DataLoader`) really need physical image files, then we must also ensure our images (which sometimes are `AlternativeImage` and sometimes generated by core, but never with a file path) are serialised...